src/server/ua_server.c

```diff
@@ -701,10 +706,9 @@ secureChannel_delayedCloseTrustList(void *application, void *context) {
     UA_CertificateGroup certGroup = server->config.secureChannelPKI;
     UA_SecureChannel *channel;
     TAILQ_FOREACH(channel, &server->channels, serverEntry) {
-        const UA_SecurityPolicy *policy = channel->securityPolicy;
         if(channel->state != UA_SECURECHANNELSTATE_CLOSED && channel->state != UA_SECURECHANNELSTATE_CLOSING)
             continue;
-        if(certGroup.verifyCertificate(&certGroup, &policy->localCertificate) != UA_STATUSCODE_GOOD)
+        if(certGroup.verifyCertificate(&certGroup, &channel->remoteCertificate) != UA_STATUSCODE_GOOD)
```

Comment:
Do we need to fix that for 1.4 also?

Reply:
No. This is only relevant for the master.